libraries/AP_Camera/AP_Camera.cpp

```diff
@@ -55,10 +55,27 @@ const AP_Param::GroupInfo AP_Camera::var_info[] PROGMEM = {
     // @Values: 0:Low,1:High
     // @User: Standard
     AP_GROUPINFO("RELAY_ON",    5, AP_Camera, _relay_on, 1),
-    
+
+    // @Param: MIN_INTERVAL
+    // @DisplayName: Minimum time between photos
+    // @Description: Postpone shooting if previous picture was taken less than preset time ago.
+    // @User: Standard
+    // @Units: milliseconds
+    // @Range: 0 10000
+    AP_GROUPINFO("MIN_INTERVAL",  6, AP_Camera, _min_interval, 0),
+
+    // @Param: MAX_ROLL
+    // @DisplayName: Maximum photo roll angle.
+    // @Description: Postpone shooting if roll is greater than limit.
+    // @User: Standard
+    // @Units: Degrees
+    // @Range: 0 180
+    AP_GROUPINFO("MAX_ROLL",  7, AP_Camera, _max_roll, 45),
```

Comment:
true, for some reason I needed to use (I did not actually see it was signed)
AP_Int8         _max_roll;   
uint8_t caused would not compile I think.
I agree that the default of 180 as "disabled" is not very intuitive, but it leaves to option to trigger only within 0.999 degree, if 0=disabled then mimimum acceptable roll would be 1.999degree.
I could improve the comment to make it clear that 180=disabled, but then again, it's the default, and those who tweak it will know how it works.